requirements.adoc

```diff
@@ -207,12 +269,30 @@ requested procedure for reporting and create the report.
 COMPONENTS::
 Its main components are
 
-* corresponding requested procedure
```

Comment:
would not delete the requested procedure. its what links everything together with the order. (and also defined in IHE RAD this way)